src/main.js

```diff
@@ -78,6 +79,7 @@ const store = new Vuex.Store({
   state: {
     user: null,
     scope: new Scope(),
+    fluent: new Fluent(),
```

Comment:
The state is usually reserved for storing application state. I don't know Fluent too well, but I'm wondering if a better place for the Fluent instance might be just another Fluent.js file that we import from in the rest of the application. I can't quite tell if this is implemented via a technical necessity, but I'm just wondering if that's possible.

Reply:
This was one of the things I wanted to discuss: so, we have to do some setup with fluent (which you'll see in the `Fluent` constructor), like parsing the string files and constructing the bundle out of it. My understanding is, if we import and initialize a `Fluent` class in every file we want to use it (which will eventually be close to all of them), we'll be doing this setup multiple times, for no real reason. We might also want to cache computed translated strings in the future.

Maybe there's some better way of doing "initialize once, use everywhere" in Vue? I just followed the existing pattern we're using for `Scope`.

Reply:
Oh yea, that makes a lot of sense. So setting up the Fluent instantiating in one file and then importing it every other file would be the way to handle that. It doesn't reinstantiate the file on each import.
I'm doing that actually for the store and the router in my PR: https://github.com/mozilla-iam/dino-park-front-end/pull/505
You can prove this to yourself by putting a console.log in the "fetcher.js" file which is being pulled in by multiple components and confirm that file gets run only once.

The difference between this and the Scope constructor is that the Scope is actually storing the state of the user if you look at the scope object. So it makes a little more sense than storing a library.
Super cool stuff! Thanks for getting this in our codebase!

Reply:
I still want to avoid having to import fluent in every file (because we'll eventually be using it in close to every one), but yes agreed that it makes no sense to be in the vuex store. I've achieved that by just initializing Fluent as a const in `main.js` and still referencing it in the mixin method.